src/components/forms/validator.js

```diff
@@ -88,7 +88,10 @@ export const uniqueAddress = (addresses: string[] | List<string>) =>
     return addressAlreadyExists ? ADDRESS_REPEATED_ERROR : undefined
   })
 
-export const composeValidators = (...validators: Function[]): FieldValidator => (value: Field, values, meta) => {
+export const composeValidators = (...validators: Function[]): FieldValidator => (value: Field) =>
+  validators.reduce((error, validator) => error || validator(value), undefined)
+
+export const composeValidatorsApps = (...validators: Function[]): FieldValidator => (value: Field, values, meta) => {
```

Comment:
This shouldn't break the validation in safe creation. If the field is not modified, the form should be invalid anyways. Have you tied to disable the submit button when form is invalid?

Reply:
If there is not way to make it work without duplicating this code, we should find a better name.

Reply:
I've modified the code to avoid the duplication of the code and make `composeValidatorsApps ` a wrapper of the old `composeValidators `. I don't wanna modify a lot the logic because it will require a lot of testing in all the forms that use those functions and. we are going to do a release. I've also tried to disable the next button but it will complicate the logic more than this wrapper does